app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -24,81 +26,87 @@ class ProjectBoard extends React.Component {
     this.props.fetchProjectBoard(this.props.projectId);
   }
 
+  onAddStory = (status) => {
+    const {createStory} = this.props;
+    return createStory({status});
+  }
+
+  renderColumnAction = (status) =>
+    <AddStoryButton onAdd={() => this.onAddStory(status)}/>
+
+
+  getColumnTitle = () => {
```

Comment:
You can create a getter function using `get` here:
```js
get columnTitle() {
  const { history } = this.props;
  return [I18n.t("projects.show.history"), "'", history.storyTitle, "'"].join(' ');
}
```
And then, access this value like this: `this.columnTitle` 😄 
`title={() => this.columnTitle}`

Reply:
Can you explain why use a getter??

Reply:
Usually is good to use getter implement when their result is derived from the component's props/state and you don't need to pass any parameter.
Here is a simple explanation/example from [Stackoverflow](https://stackoverflow.com/a/41518168/4538055)  😄

Reply:
Thanks, gotcha!

Reply:
Thanks, gotcha!